### ISSUE TITLE
TypeScript 4.4 and removed `MouseWheenEvent`

### ISSUE THREAD

**Issue by jvanoostveen** [AFASSoftware/maquette#173]
When running our codebase with TypeScript 4.4 beta, I noticed the following error:

```
node_modules/maquette/dist/interfaces.d.ts:215:36 - error TS2304: Cannot find name 'MouseWheelEvent'.
215     onmousewheel?(ev: WheelEvent | MouseWheelEvent): boolean | void;
```

It seems TypeScript removed the non-standard deprecated [MouseWheelEvent](https://developer.mozilla.org/en-US/docs/Web/API/MouseWheelEvent) from its lib.

**Comment by jvanoostveen** [comment]
A workaround is adding
```
interface MouseWheelEvent extends WheelEvent {}
```
to our custom / backwards compatibility `extensions.d.ts`.

**Comment by jvanoostveen** [comment]
Resolved in upcoming release (3.4.2)

**Comment by jvanoostveen** [comment]
Correction, v3.5.1
